docs/src/main/paradox/assets/api-reference/domains/existing-domain.json

```diff
@@ -1,8 +1,10 @@
 {
-  "@context": {
-    "nxv": "https://nexus.example.com/v0/voc/nexus/core/",
-    "schema": "http://schema.org/"
-  },
+  "@context": [
```

Comment:
th order should be opposite right, first currently defined context and afterwards the resource context link. I think it is wrong order everywhere else too.